### PR TITLE
change cluster creation log message

### PR DESCRIPTION
**Pull request by nicolai86**
## What does this PR do?

This PR changes the messaging when a BYOC cluster is created to help guide the users.

See LAI2-10481 for details.

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: src/lightning_app/cli/cmd_clusters.py [hunk 1]
```diff
@@ -91,7 +91,16 @@ def create(
         if wait:
             _wait_for_cluster_state(self.api_client, resp.id, V1ClusterState.RUNNING)
 
-        click.echo(f"{resp.id} cluster is in {resp.status.phase} state")
+        click.echo(
+            f"""{resp.id} is now being created... This can take up to an hour.
+
+        To view your clusters status use:
```
Comment (awaelchli) [inline src/lightning_app/cli/cmd_clusters.py:97]: Is so much indentation intentional?

Comment (nicolai86) [reply src/lightning_app/cli/cmd_clusters.py]: it should not - changed in subsequent commit.

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by awaelchli** [comment]
Package build is troubled by syntax error:
`src/lightning/app/cli/cmd_clusters.py:6: [E999] SyntaxError: invalid syntax`
Is it related to the meta-package logic? @Borda

**Review by kaushikb11 — APPROVED** [review]

File: src/lightning_app/cli/cmd_clusters.py [hunk 2]
```diff
@@ -136,7 +136,16 @@ def create(
         if wait:
             _wait_for_cluster_state(self.api_client, resp.id, V1ClusterState.RUNNING)
 
-        click.echo(f"{resp.id} cluster is in {resp.status.phase} state")
+        click.echo(
+            f"""{resp.id} is now being created... This can take up to an hour.
+
+To view your clusters status use:
```
Comment (Felonious-Spellfire) [inline src/lightning_app/cli/cmd_clusters.py:142]:
```suggestion
To view the status of your clusters use:
```

**Comment by nicolai86** [comment]
> Package build is troubled by syntax error: `src/lightning/app/cli/cmd_clusters.py:6: [E999] SyntaxError: invalid syntax` Is it related to the meta-package logic? @Borda

I worked around this by changing indentation and using dedent to remove the spaces. It seems the triple quotes with different endings only works in some python versions/ platforms.

**Pull request merged**